Community_Guidelines.md

```diff
@@ -0,0 +1,380 @@
+**Owner:** Mac Admins Foundation
+**Review Cycle:** Annually
+
+## Purpose
+
+These Community Guidelines supplement our [Code of Conduct](./Code_of_Conduct.md) with practical guidance for participating in The Mac Admins Slack. While the Code of Conduct defines what is not acceptable, these guidelines describe what good community participation looks like.
+
+## Getting Started
+
+### Joining the Workspace
+
+- Read and understand the [Code of Conduct](./Code_of_Conduct.md)
+- Complete your profile with your name and role (optional but encouraged)
+- Introduce yourself in #introductions
+- Browse channel descriptions to find relevant discussions
+
+### Setting Up Your Profile
+
+**Recommended profile information:**
+- Full name (or preferred name)
+- Current role and organization (optional)
+- Pronouns (optional)
+- Time zone (helpful for async communication)
+- Profile photo (optional)
+
+**Privacy note:** Only share what you're comfortable making public.
+
+## Communication Best Practices
+
+### General Guidelines
+
+**Be Clear and Concise**
+- Threading varies by channel - most channels don't thread as we predate that feature
+- Observe channel norms before posting
+- Break long messages into paragraphs
+- Use formatting (bold, code blocks) appropriately
+- Search before asking - your question may be answered already
+
+**Be Respectful of Time**
+- Don't use @channel or @here unless absolutely necessary
+- Use @admins only for urgent admin matters
+- Respect "do not disturb" and timezone differences
+- Be patient waiting for responses - this is an async community
+
+**Be Inclusive**
+- Remember this is a global community with varying English proficiency
+- Avoid idioms and cultural references that may not translate
+- Be patient with misunderstandings
+- Offer clarification when asked
+
+### Technical Discussions
+
+**Asking Questions**
+- Search Slack history first
+- Provide context (OS version, device model, MDM, etc.)
+- Include error messages or logs (use snippets for long text)
+- Post in the most relevant channel
+- We're here to educate and help you learn, not do your work for you
+- Update the thread if you solve your own problem
+
+**Answering Questions**
+- Be kind - everyone was new once
+- Explain your reasoning, don't just post commands
+- Link to documentation when relevant
+- If you're unsure, say so
+- Follow the channel's threading conventions
+
+**Sharing Solutions**
+- Share code snippets as files or formatted code blocks
+- Include comments in scripts
+- Explain any prerequisites or caveats
+- Link to GitHub repos for longer code
+- Credit original authors if adapting their work
+
+### Debates and Disagreements
+
+**Healthy Debate**
+- Focus on ideas, not people
+- Use "I" statements: "I've found..." not "You're wrong..."
+- Acknowledge valid points
+- Know when to agree to disagree
+- Consider moving heated discussions to DM
+
+**When Tensions Rise**
+- Take a break before responding
+- Assume good intent
```

Comment:
I agree we should always assume good intent, but this doesn't address that impact matters regardless of intent. Someone can cause harm even with good intentions. Consider balancing this with language about taking responsibility for impact.